packages/build/src/steps/get.js

```diff
@@ -7,15 +8,18 @@ import { saveArtifacts } from '../plugins_core/save_artifacts/index.js'
 import { scanForSecrets } from '../plugins_core/secrets_scanning/index.js'
 
 // Get all build steps
-export const getSteps = function (steps) {
+export const getSteps = function (steps, eventHandlers) {
   const stepsA = addCoreSteps(steps)
-  const stepsB = sortSteps(stepsA, EVENTS)
+  const eventSteps = getEventSteps(eventHandlers)
```

Comment:
nit: I think `eventHandlerSteps` would be a bit more descriptive

Reply:
💯